subprojects/rizin-shell-parser/grammar.js

```diff
@@ -268,6 +268,7 @@ module.exports = grammar({
           ),
           seq(field("command", alias(/\.[ ]+/, $.cmd_identifier)), field("args", optional($.args))),
           seq(field("command", alias(".(", $.cmd_identifier)), field("args", $.macro_call_content)),
+          seq(field("command", alias("..(", $.cmd_identifier)), field("args", $.macro_call_content)),
```

Comment:
please add a test in the corpus of rizin-shell-parser

Reply:
okay fdd7d0795a37f98a0a906d3fab1394aa48394630 ... I've pretty-printed the s-expression partially using Emacs' `indent-pp-sexp` and partially by hand -- is there a more efficient way?